sklearn/tests/test_pipeline.py

```diff
@@ -418,6 +418,13 @@ def test_make_union():
     assert_equal(transformers, (pca, mock))
 
 
+def test_make_union_kwargs():
+    pca = PCA(svd_solver='full')
+    mock = Transf()
+    fu = make_union(pca, mock, n_jobs=3)
+    assert_equal(3, fu.n_jobs)
```

Comment:
you might, for completeness, check that `fu.transformers == make_union(pca, mock).transformers` but this is fine as is.